Solutions/SonraiSecurity/Data Connectors/Connector_REST_API_Sonrai.json

```diff
@@ -0,0 +1,104 @@
+{
+  "id": "SonraiDataConnector",
+  "title": "Sonrai Data Connector",
+  "publisher": "Sonrai",
+  "descriptionMarkdown": "Use this data connector to integrate with Sonrai Security and get Sonrai tickets sent directly to Azure Sentinel.",
+  "graphQueries": [
+    {
+      "metricName": "Sonrai Tickets",
+      "legend": "Sonrai_Tickets_CL",
+      "baseQuery": "Sonrai_Tickets_CL"
+    }
+  ],
+  "sampleQueries": [
+    {
+      "description" : "Query for tickets with PermissionList violations",
+      "query": "Sonrai_Tickets_CL \n| where evidence_policyEvidence_label_s == \"PermissionList\"\n| limit 10"
```

Comment:
@Cameron-Dahr-sonrai, Given sample query is failing at column 'evidence_policyEvidence_label_s' , Can you please check and update? Thanks!
![image](https://user-images.githubusercontent.com/75478517/129705419-3afc18d1-df68-40fa-8f54-0bb2e93628ed.png)

Reply:
Modified this query